demisto_sdk/scripts/init_validation_script.py

```diff
@@ -452,8 +452,8 @@ def generate_supported_content_types_section(self):
                 for content_type in self.content_types
             ]
             self.supported_content_types = (
-                f"ContentTypes = Union{str(supported_content_types)}"
-            )
+                f"ContentTypes = Union{(supported_content_types)}"
+            ).replace("'", "")
```

Comment:
whats this change?

Reply:
Putting str on the whole list would print the content types with "'" around them, and we need the actual object rather than the string of its class name.